setup.py

```diff
@@ -220,7 +220,7 @@
             'inspirehep_holding_js = inspirehep.modules.workflows.bundles:details_js',
         ],
         'invenio_jsonschemas.schemas': [
-            'inspire_records = inspirehep.modules.records.jsonschemas',
+            'inspire_records = inspire_schemas',
```

Comment:
Yep, it was easy :smiley:, just needed the proper packaging on the inspire_schemas side.